iceoryx_utils/platform/win/include/iceoryx_utils/platform/platform_correction.hpp

```diff
@@ -11,7 +11,8 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
+#ifndef IOX_UTILS_PLATFORM_PLATFORM_CORRECTION_HPP
+#define IOX_UTILS_PLATFORM_PLATFORM_CORRECTION_HPP
 
 // !NO PRAGMA ONE HERE! we want that these macros are always executed and
 // not just once, otherwise the windows macros are kicking in and nothing
```

Comment:
Update comment and remove include guard? Have you checked both Windows and Mac builds?

Reply:
Missed that one while the script was processing all includes. Looks like an include guard cannot be used here. I only compiled on Linux after the modifications.